src/main/resources/add-ons-to-index.json

```diff
@@ -645,7 +666,7 @@
       "type": "OMOD",
       "name": "Database Messages",
       "tags": [
-        "administration"
+        "administration", "Database"
```

Comment:
"database" is not a helpful tag.

Reply:
@f4ww4z are you sure it is not helpful for a database backup module?

Reply:
@dkayiwa I am sure as @djazayeri also suggests it in another PR. See the comment on  https://github.com/openmrs/openmrs-contrib-addonindex/pull/27#discussion_r154514608

Reply:
Yes @dkayiwa . @Yashu-Jindal You will need to remove "database" since grouping modules with the tag doesn't suggest a meaningful purpose.